src/components/subPages/taskData/taskData.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {ChangeEvent, useState} from 'react'
+import './taskData.scss'
+import {Link, useParams} from 'react-router-dom'
+import TaskDataError from '../../errorSubpage/taskDataError'
+import {useMutation, useQuery, useQueryClient} from 'react-query'
+import LoadingAnimation from '../../animations/loadingAnimation'
+import {toast} from 'react-toastify'
+import requestTaskApi from '../../axiosConfig'
+
+const TaskData = () => {
+
+	const {id} = useParams()
+	const queryClient = useQueryClient()
+
+	const {isLoading, data} = useQuery(['task', `${id}`],  () =>
+		requestTaskApi.get(`/api/tasks/${id}`),
+	{
+		refetchOnWindowFocus: false

```

Comment:
problem będzie występował we wszystkich query, więc znajdź rozwiązanie globalne na wsyzstkie query